static/app/views/settings/organizationAuditLog/index.tsx

```diff
@@ -69,64 +70,96 @@ const EVENT_TYPES = [
   'sentry-app.uninstall',
 ];
 
-type Props = RouteComponentProps<{orgId: string}, {}> &
-  AsyncView['props'] & {
-    organization: Organization;
-  };
+type Props = {
+  organization: Organization;
+};
 
-type State = AsyncView['state'] & {
+type State = {
   entryList: AuditLog[] | null;
   entryListPageLinks: string | null;
+  isLoading: boolean;
+  currentCursor?: string;
+  eventType?: string;
 };
 
-class OrganizationAuditLog extends AsyncView<Props, State> {
-  getEndpoints(): ReturnType<AsyncView['getEndpoints']> {
-    return [
-      [
-        'entryList',
-        `/organizations/${this.props.params.orgId}/audit-logs/`,
-        {
-          query: this.props.location.query,
-        },
-      ],
-    ];
-  }
-
-  getTitle() {
-    return routeTitleGen(t('Audit Log'), this.props.organization.slug, false);
-  }
+function OrganizationAuditLog({organization}: Props) {
+  const [state, setState] = useState<State>({
+    entryList: [],
+    entryListPageLinks: null,
+    isLoading: true,
+  });
 
-  handleEventSelect = (value: string) => {
-    // Dont update if event has not changed
-    if (this.props.location.query.event === value) {
-      return;
-    }
+  const api = useApi();
 
-    browserHistory.push({
-      pathname: this.props.location.pathname,
-      search: `?event=${value}`,
-    });
+  const handleCursor: CursorHandler = resultsCursor => {
+    setState(prevState => ({
+      ...prevState,
+      currentCursor: resultsCursor,
+    }));
   };
 
-  renderLoading() {
-    return this.renderBody();
-  }
+  const fetchAuditLogData = useCallback(
+    async (cursor = state.currentCursor, event = state.eventType) => {
```

Comment:
If you're never calling this method with parameters you could remove these args and create `payload` as a simple object.